Document Snaps homepages (#1082)
MetaMask/metamask-docs: snaps/reference/permissions.md

those connected accounts.
:::

### endowment:page-home

:::flaskOnly
:::

To present a dedicated UI within MetaMask, a Snap must request the `endowment:page-home` permission. 

> This permission allows the Snap to specify a "home page" entrypoint with the exported 
> [`onHomePage`](../reference/exports.md#onhomepage) method. 
> This "home page" can use any [custom UI components](../how-to/use-custom-ui.md) to build an embedded page accessible via the Snaps menu. 

Reviewer: ```suggestion
This permission allows the Snap to specify a "home page" entrypoint using the exported 
[`onHomePage`](../reference/exports.md#onhomepage) method. 
You can use any [custom UI components](../how-to/use-custom-ui.md) to build an embedded home page accessible through the Snaps menu.
```
